scripts/handlers/subtract.js

```diff
@@ -83,10 +83,26 @@ function subtractHandler() {
 
   // read a number from the user
 
+  const input = prompt('Write Some Number');
+  if (input === null || input === "") {
+    alert('Please enter a valid number');
+  } else {
+    let inputConfirmed = +input;
+    if (Object.is(inputConfirmed, NaN)) { alert('Be sure you are entering a number! Try again. '); }
+    else {
+      // subtract the last result by the user's number and reassign lastResult
+      const result = subtract(inputConfirmed, lastResult);
```

Comment:
Hi Abel, 
there is a shared variable called 'lastResult'. Here in this line, the variable 'lastResult' should be reassigned instead creating a new const variable 'result'. And also the lastResult should be subtracted from the user's input(just reverse the arguments inside the function).

Thank you